components/ActionSheet/examples/ActionSheetSwiftExample.swift

```diff
@@ -204,4 +207,16 @@ extension ActionSheetSwiftExample {
     actionSheet.addAction(actionThree)
     return actionSheet
   }
+
+  static func thirtyOptions() -> MDCActionSheetController {
+    let actionSheet = MDCActionSheetController(title: "Action sheet", message: messageString)
+    for i in 1...30 {
+      let action = MDCActionSheetAction(title: "Action \(i)",
+                                        image: UIImage(named: "Home")!,
```

Comment:
instead of force unwrapping check outside the for loop if this image exists

Reply:
`guard` or `if let`

Reply:
If we do a check outside the loop then it will force us to have two `for` loops, instead I removed the `!` because the image can be `nil`.